src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -371,9 +371,10 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
   <data name="AddPkg_FrameworksDescription" xml:space="preserve">
     <value>Frameworks for which the package reference should be added.</value>
   </data>
-  <data name="AddPkg_MissingArgument" xml:space="preserve">
-    <value>Unable to add package Reference, argument '{0}' not provided. Please use -h|--help for help.</value>
-    <comment>{0} - Argument which was not provided</comment>
+  <data name="Error_PkgMissingArgument" xml:space="preserve">
+    <value>Unable to '{0}' package Reference, argument '{1}' not provided. Please use -h|--help for help.</value>
```

Comment:
I wouldn't add the ``-h|--help`` here. Users should know this, also this might end up getting localized and turn into an invalid parameter.